aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -358,6 +358,54 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
 
 The default value is 2^14 (16,384).
 
+### Maximum request header size
+
+`Http2.MaxRequestHeaderFieldSize` indicates the maximum allowed size in octets of request header values. This limit applies to both name and value together in their compressed and uncompressed representations. The value must be greater than zero (0).
+
+```csharp
+public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+    WebHost.CreateDefaultBuilder(args)
+        .UseStartup<Startup>()
+        .ConfigureKestrel((context, options) =>
+        {
+            options.Limits.Http2.MaxRequestHeaderFieldSize = 12288;
```

Comment:
For consistency with the other examples, let's either:

* Make these code samples match the default (8192) in this case. `131072` in the next sample. `98304` in the third.
* Make all of the examples use values different from their defaults. I can open an issue later to do that if you prefer to go this way ... no need to do it on this PR.